src/main/flight/feedforward.c

```diff
@@ -153,6 +154,10 @@ FAST_CODE_NOINLINE float feedforwardApply(int axis, bool newRcFrame, feedforward
         } else {
             setpointDelta[axis] = setpointDeltaImpl[axis];
         }
+
+        // apply feedforward transition
+        setpointDelta[axis] *= ffTransitionFactor > 0 ? MIN(1.f, getRcDeflectionAbs(axis) * ffTransitionFactor) : 1;
```

Comment:
For better readability I suggest `1.0f` instead of `1.f` (note the compiler does not care):
```suggestion
        setpointDelta[axis] *= ffTransitionFactor > 0 ? MIN(1.0f, getRcDeflectionAbs(axis) * ffTransitionFactor) : 1;
```